oat-sa/extension-tao-testqti
# Feature/AUT-1427/subsection indexation

Related to: https://oat-sa.atlassian.net/browse/AUT-1427

Depends on:

- [ ] https://github.com/oat-sa/tao-core-sdk-fe/pull/128

How to test:

- go to test Authoring
- add sections, subsections, items
- try to move, delete
- indexes should be properly updated
